### PR TITLE
[website] Add new committers

### PR DESCRIPTION
**Pull request by Jennifer88huang**
### Motivation

We have two more committers for Pulsar community.

### Modifications

Add their info on the website.

### CODE REVIEW

File: site2/website/data/team.js [hunk 1]
```diff
@@ -37,12 +37,24 @@ module.exports = {
       apacheId: 'guangning',
       roles: 'Committer'
     },
+    {
+      name: 'Hang Chen',
+      apacheId: 'chenhang',
+      org: 'Bigo Technology',
```
Comment (Huanli-Meng) [inline site2/website/data/team.js:43]:
```suggestion
      org: 'BIGO Technology',
```

image

**Review by Huanli-Meng — COMMENTED** [review]
Leave a comment, PTAL

**Review by Huanli-Meng — APPROVED** [review]

**Comment by hangc0276** [comment]
+1  Thanks.

**Comment by Jennifer88huang** [comment]
@hangc0276 @Huanli-Meng Congratulations to you~

**Review by codelipenghui — APPROVED** [review]

**Pull request merged**